index.html

```diff
@@ -0,0 +1,248 @@
+                            Sponsor
+                        </a>
+                    </li>
+                    <li class="nav-item px-5 px-md-1">
+                        <a href="#" class="nav-link text-secondary">
+                            News
+                        </a>
+                    </li>
+                    <li class="nav-item px-5 px-md-2">
+                        <button type="button" class="btn" id="btn-header">Campaing</button>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <section id="welcome-section"  class="d-flex flex-column py-5 ps-4 py-md-3 ps-md-5">
+        <div class="ms-md-5">
+            <h1 class="lato-font">"Hello! Calisthenics world"</h1>
+        </div>
+        <div id="welcome-info" class="ms-md-5 mb-md-4">
+            <h2 class="oswald-font">
+                Profesional Calisthenics Exposition CDMX 2021
+            </h2>
+        </div>
+        <div id="definition-info" class="mb-3 ms-md-5">
+            <p>
+            The biggest names in calisthenics will be at the expo, promoting a healthy lifestyle. Also,
+            we're hosting a 16 man single elimination tournament that has the goal
+            of finding the top new athletes around the world. in CDMX
+            </p>
+        </div>
+        <div id="date-info" class="ms-md-5">
+            <p class="lato-font">
+                <span class="oswald-font">2021.11.11(THU) &nbsp; ~ &nbsp; 12(FRI)</span><br>
+                 <br> @ EXPO SANTA FE MÉXICO, VASCO DE QUIROGA 3000
+            </p>
+        </div>
+    </section>
+    <section id="main-program" class="d-flex justify-content-center align-items-center">
+        <div class="d-flex align-items-center flex-column">
+            <h2 class="text-white mb-0 mt-4">Main Program</h2>
+            <hr class="half-border">
+        </div>
+        <div class="container">
+            <div class="row row-cols-1 row-cols-md-4 gx-4 gx-md-3 gy-2">
+                <div class="col">
+                    <div class="card">
+                        <div class="card-body">
+                            <ul class="d-flex program-cards">
+                                <li>
+                                    <i class="fas fa-book"></i>
+                                </li>
+                                <li>
+                                    <h3 class="text-danger">Lecture</h3>
+                                </li>
+                                <li>
+                                    <p>Listen to our sponsors from various countries, about the newest and best equipment.</p>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+                <div class="col">
+                    <div class="card">
+                        <div class="card-body">
+                            <ul class="d-flex program-cards">
```

Comment:
[Optional]
On the design template, this section has 5 articles. It will be better if you add one more.